zerver/openapi/zulip.yaml

```diff
@@ -13051,6 +13055,13 @@ components:
           description: |
             The path relative to the organization's URL where the
             emoji's image can be found.
+        still_url:
+          type: string
+          description: |
+            The path relative to the organization's URL where the
+            emoji's gif's still image can be found.
```

Comment:
This should have a couple sentences explaining the background of animated GIFs, etc.